### PR TITLE
[SW-654] cleaning transaction after change multisig account

### PR DESCRIPTION
**Pull request by konradodo**

### CODE REVIEW

**Review by ifaouibadi — APPROVED** [review]

File: src/popup/components/TransactionList.vue [hunk 1]
```diff
@@ -238,6 +238,7 @@ export default defineComponent({
     });
 
     onMounted(async () => {
+      root.$store.commit('initTransactions');
```
Comment (pstachel) [inline src/popup/components/TransactionList.vue:241]: I think that initTransaction is handled on account change - you could work on that or wait for SW-620 to be merged

File: src/popup/components/TransactionList.vue [hunk 2]
```diff
@@ -214,42 +215,28 @@ export default defineComponent({
       }
     }
 
-    async function loadMore() {
+    async function loadMore(recent: boolean = false) {
       if (loading.value) return;
```
Comment (pstachel) [inline src/popup/components/TransactionList.vue:219]: if (loading.value) {
    return;
}

File: src/popup/components/TransactionList.vue [hunk 3]
```diff
@@ -214,42 +215,28 @@ export default defineComponent({
...
       checkLoadMore();
     });
 
     onMounted(async () => {
-      loadMore();
-      const polling = setInterval(() => getLatest(), 10000);
-      root.$once('hook:destroyed', () => {
-        clearInterval(polling);
-        isDestroyed.value = true;
-      });
```
Comment (pstachel) [inline src/popup/components/TransactionList.vue:247]: why did we get rid of transaction polling?

**Review by pstachel — CHANGES_REQUESTED** [review]

**Review by pstachel — APPROVED** [review]

File: src/popup/components/TransactionList.vue [hunk 4]
```diff
@@ -168,13 +170,13 @@ export default defineComponent({
...
-                      tr.tx.recipientId === currentAddress.value
-                      || (
-                        tr.tx.senderId !== currentAddress.value
-                          && tr.tx.recipientId.startsWith('nm_')
-                      )
+                  && (
+                    tr.tx.recipientId === currentAddress.value
+                    || (
+                      tr.tx.senderId !== currentAddress.value
+                      && tr.tx.recipientId.startsWith('nm_')
```
Comment (peronczyk) [inline src/popup/components/TransactionList.vue:177]: By the way you can replace this with `isAensName` utility function.

**Review by peronczyk — APPROVED** [review]

**Review by CedrikNikita — CHANGES_REQUESTED** [review]
This PR is not fixing 654 completly, it is done in #1863.
I suggest to leave only a truncateWidth fix here.

**Pull request closed without merging**